dists/install.sh

```diff
@@ -112,30 +112,43 @@ main() {
 
   version=$(curl --silent https://cli-dl.scalingo.com/version | tr -d ' \t\n')
   if [ -z "$version" ]; then
-    echo "-----> Fail to get the version of the CLI" >&2
-    echo "You probably have an old version of curl. Please check your curl version and update accordingly." >&2
+    error "Fail to get the version of the CLI\n"
+    error "You probably have an old version of curl. Please check your curl version and update accordingly.\n"
     exit 1
   fi
 
   dirname="scalingo_${version}_${os}_${arch}"
   archive_name="${dirname}.${ext}"
-  url=https://github.com/Scalingo/cli/releases/download/${version}/${archive_name}
+  url="https://github.com/Scalingo/cli/releases/download/${version}/${archive_name}"
 
   status "Downloading Scalingo client...  "
   curl --silent --fail --location --output ${tmpdir}/${archive_name} ${url}
   if [ ! -f ${tmpdir}/${archive_name} ]; then
-    echo "" >&2
-    echo "-----> Fail to download the CLI archive" >&2
+    echo ""
+    error "Fail to download the CLI archive\n"
     exit 1
   fi
   echo "DONE"
+
+  status "Verifying the checksum...  "
+  checksums_url="https://github.com/Scalingo/cli/releases/download/${version}/checksums.txt"
+  checksum_computed=$(sha256sum ${tmpdir}/${archive_name} | cut -d " " -f1)
+  checksum_expected=$(wget -q --output-document - $checksums_url | grep $archive_name | cut -d " " -f 1)
+  if [[ "$checksum_computed" != "$checksum_expected" ]]; then
+    echo "INVALID"
+    error "Checksums don't match ('$checksum_computed' != '$checksum_expected').\n"
```

Comment:
suggestion(non-blocking): As you said in the PR description, I'm not sure either to keep displaying the checksums.
Could it be possible to only display "Checksums don't match" but when adding `DEBUG=1`, then display the checksums?

Reply:
There is already a `DEBUG=true` which defines `set -x`, which would display the variables content. Hence I removed my line.